src/modules/InitApp/index.js

```diff
@@ -39,16 +39,18 @@ class InitApp {
    const chatType = data.to_group_id ? 'group_chat' : 'private_chat';
    const chatFromId = data.to_group_id ? data.to_group_id : data.from_user;
    const title = data.to_group_id && data.groupName ? data.groupName : name;
+   const audio = "http://cdnringuc.shoujiduoduo.com/ringres/user/a48/304/18687304.aac";
```

Comment:
I upload it with qi niu yun. Pls replace this URL with 'https://cdn.aermin.top/audio.aac' .